AppCenterAuth/AppCenterAuth/MSAuth.m

```diff
@@ -285,11 +325,19 @@ - (void)downloadConfigurationWithETag:(nullable NSString *)eTag {
                   // Reinitialize client application.
                   [self configAuthenticationClient];
                 }
+                [self continueSignInAndStopWaitingForConfig:YES];
               } else {
                 MSLogError([MSAuth logTag], @"Downloaded auth config is not valid.");
+                [self ifSignInExistsCompleteWithErrorCode:MSACAuthErrorSignInConfigNotValid
+                                       andMessage:@"Downloaded auth config is not valid."
```

Comment:
I don't like having to pass the message as a hard-coded string twice

Reply:
Should we construct the error outside the method and pss it?